src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -678,6 +687,16 @@ public enum NuGetLogCode
         /// </summary>
         NU5037 = 5037,
 
+        /// <summary>
+        /// Empty icon extension error
```

Comment:
Is there too much value in havign separate error codes? How common is this? Should it really take 2 codes? 

Also, it sounds like you're leaning towards a warning, remember to fix the code.